utils/test_results.py

```diff
@@ -270,7 +271,14 @@ def compare(row: TestResultsRow) -> int:
         row_value = getattr(row, ordering)
         row_value_str = str(row_value)
         cursor_value_str = cursor.ordered_value
-        return (row_value_str > cursor_value_str) - (row_value_str < cursor_value_str)
+        if descending:
+            return (row_value_str < cursor_value_str) - (
```

Comment:
Is there a way we can find a name for `row_value_str < cursor_value_str` and `row_value_str > cursor_value_str` such that we can have something like

varOne - varTwo if descending else varTwo - varOne

It took me a sec to realize that the terms in each equation were the same, just flipped